guides/advanced-use/project-layouts.md

```diff
@@ -8,18 +8,18 @@ For example, if you have a `post` resource and a `tags` component in your projec
 ```shell
 app
 ├── components
-|   └── tags.js
+|   ├── tags.js
+|   └── tags.hbs
 ├── controllers
 |   └── post.js
 ├── models
 |   └── post.js
 ├── routes
 |   └── post.js
 ├── templates
-|   ├── components/tags.hbs
 |   └── post.hbs
 └── app.js
-└── index.html
+├── index.html
 └── resolver.js
 └── router.js
```

Comment:
@lifeart can you fix these as well (`app.js` has the same problem, as well as some of the files in the pods section)